libraries/AC_AttitudeControl/AC_AttitudeControl_Logging.cpp

```diff
@@ -45,8 +45,8 @@ void AC_AttitudeControl::Write_Rate(const AC_PosControl &pos_control) const
         control_yaw     : rate_targets_degs.z,
         yaw             : gyro_rate.z,
         yaw_out         : _motors.get_yaw()+_motors.get_yaw_ff(),
-        control_accel   : (float)accel_target_neu_cmss.z,
-        accel           : (float)(-(_ahrs.get_accel_ef().z + GRAVITY_MSS) * 100.0f),
+        control_accel   : (float)(-accel_target_ned_mss.z),
+        accel           : (float)(-(_ahrs.get_accel_ef().z + GRAVITY_MSS)),
```

Comment:
I guess the change to the accel logging is an existing bug?  As a side note, it's annoying that the AP_AHRS::get_accel_ef() method has no comment above it stating the frame and units